src/components/Pagination/README.md

```diff
@@ -0,0 +1,36 @@
+## Pagination
+An implementation of Pagination for TPAs.
+The TPA implementation provides few extendable theme properties with the ability to override the base properties as well,
+
+## Theme properties (All are optional)
+
+| propName   | propType | defaultValue | description |
+|------------|----------|--------------|-------------|
+| TextFont | string   | Palette font_8 | The font of the pagination pages |
+| TextFontColor  | string   | #000 | The color of the pagination pages |
+| SelectedTextFontColor  | string   | #ed24d9 | The color of the selected page |
```

Comment:
As stated before, these should be template colors.
I'm checking with the designers which colors they want them to be.